app/controllers/payment-links/metadata/resource-controller.js

```diff
@@ -0,0 +1,36 @@
+const { response } = require('../../../utils/response')
```

Comment:
Would be useful to have tests for this controller ?

Reply:
I think so! it's currently used by every controller though and don't think it fits into the scope of this PR. We could do this is a BAU improvement (although I would like to stop passing `req`, `res` etc. down into this library and refactor it to just provide a context).